Remotion/Core/UnitTests/Collections/DictionaryExtensionsTest.cs

```diff
@@ -26,7 +26,7 @@ namespace Remotion.UnitTests.Collections
   [TestFixture]
   public class DictionaryExtensionsTest
```

Comment:
probably wants some NRT tests

Reply:
Are NRT tests in general something to fix in this PR or a separate ticket?

Reply:
I think a separate ticket would be okay for this. we just should try and do them sometime during the summer